appveyor.yml

```diff
@@ -15,12 +15,7 @@ environment:
   TOOL_NODE_FLAGS: --expose-gc
   TIMEOUT_SCALE_FACTOR: 8
   METEOR_HEADLESS: true
-  SELF_TEST_EXCLUDE: "\
-      ^old cli tests|\
-      ^minifiers can't register non-js|\
-      ^minifiers: apps can't use|\
-      ^compiler plugins - addAssets|\
-      ^NULL-LEAVE-THIS-HERE-NULL$"
+  SELF_TEST_EXCLUDE: "^NULL-LEAVE-THIS-HERE-NULL$"
```

Comment:
Same question about removing here.

(The `^NULL-LEAVE-THIS-HERE-NULL$` was, IIRC, to prevent odd command line behavior exhibited when no tests were defined).

Reply:
Same as above comment - I guessed that was the reason for `NULL-LEAVE-THIS-HERE-NULL`, so I left it in place. Thanks!